### DISCUSSION TITLE
Dump current JQL query

### DISCUSSION THREAD

**Discussion by Steven-M-Brown** [ankitpokhrel/jira-cli#755] [Ideas]
I tend to use the tool to search/filter issues and it would be really handy if there was a way I could copy the jql command it is currently using to the clipboard so that I can easily share the list with people that don't use the cli without having to convert it myself.

Apologies if it already exists, I didn't see it mentioned in any of the discussions or issues

**Comment by ankitpokhrel** [comment]
There isn't a separate flag for this but `--debug` flag displays JQL along with some other info. It may not exactly be what you are looking for but could be helpful for now?

PR is welcome btw :)
